### DISCUSSION TITLE
Error adding profile under Unix System Services for Zowe Explorer

### DISCUSSION THREAD

**Discussion by excel-guy** [microsoft/vscode-discussions#650] [Extension Development QnA] [answered]
I keep getting an error setting up for a profile for use with IBM Zxplore.  I've double checked all of the profile settings & variables, and verified my credentials but get this result when I attempt to apply a User# filter onto it:

This is the error readout:   Retrieving response from uss-file-list Error: z/OSMF REST API Error: | Rest API failure with HTTP(S) status 500 | category: 1 | rc: 4 | reason: 6 | message: Path name is not valid | |

**Comment by gjsjohnmurray** [comment, marked as answer]
Please raise this on the support channel(s) of the extension(s) through which you are connecting.

This forum is intended for people who are developing extensions.
